### PR TITLE
Add teacher command 2nd revision

### PR DESCRIPTION
**Pull request by amoschee**

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -127,4 +128,8 @@ public boolean equals(Object other) {
     public int hashCode() {
         return persons.hashCode();
     }
+
+    public void addTeacher(Teacher teacher) {
+        addPerson(teacher);
```
Comment (AgentHagu) [inline src/main/java/seedu/address/model/AddressBook.java:133]: Missing JavaDoc for this public method?

**Review by AgentHagu — CHANGES_REQUESTED** [review]
Most of the code looks good, just the one nitpick to fix

File: src/main/java/seedu/address/model/AddressBook.java (hunk 1 above)
Comment (amoschee) [reply src/main/java/seedu/address/model/AddressBook.java:137]: noted, will add  javadoc for this in my next commit

**Review by AgentHagu — APPROVED** [review]
LGTM

**Pull request merged**